models/Document.php

```diff
@@ -597,7 +594,7 @@ public function getChildren(bool $includingUnpublished = false): array
                 $list->setCondition('parentId = ?', $this->getId());
                 $list->setOrderKey('index');
                 $list->setOrder('asc');
-                $this->children[$cacheKey] = $list->load();
+                $this->children[$cacheKey] = $list;
             } else {
                 $this->children[$cacheKey] = [];
```

Comment:
@dvesh3 The return type doesn't allow an array. Should be also a listing. Or should we return null if no Listing is possible?